doc/_extensions/zephyr/doxyrunner.py

```diff
@@ -132,6 +135,7 @@ def process_doxyfile(
     fmt: bool = False,
     fmt_pattern: Optional[str] = None,
     fmt_vars: Optional[Dict[str, str]] = None,
+    outdir_var: Optional[str] = None,
```

Comment:
Should this be `Optional[Path]`?

Reply:
No, since it is a variable name, not its content (`DOXY_OUT` in this case). I know it's not intuitive...